supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemJobCachingActorHelper.scala

```diff
@@ -19,5 +19,11 @@ trait SharedFileSystemJobCachingActorHelper extends StandardCachingActorHelper {
     override lazy val sharedFileSystemConfig: Config = {
       configurationDescriptor.backendConfig.as[Option[Config]]("filesystems.local").getOrElse(ConfigFactory.empty())
     }
+
+    // cachedCopyDir should be on the same physical filesystem as the execution root.
+    // WDL workflow names may not contain '-' so using 'cached-inputs' will certainly
+    // not collide with any workflows in the root directory.
+    override lazy val cachedCopyDir: Option[Path] = Some(
```

Comment:
Please use `Option` rather than `Some` throughout these changes to construct `Option` instances. This is the convention in Cromwell for null-safety even if it doesn't actually make a difference in this particular case. 😉